src/main/webapp/script.js

```diff
@@ -10,4 +10,25 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function getYouTubeComments() { 
```

Comment:
What is exactly is the end goal of this function? Why does the front end need the comments? Shouldn't we just need the statistics?